src/index.js

```diff
@@ -140,6 +140,23 @@ class ServerlessAppsyncPlugin {
     };
   }
 
+  getDeltaSyncConfig(config) {
+    if (config && config.deltaSyncConfig) {
+      if (!config.deltaSyncConfig.deltaSyncTableName) {
+        throw new Error('You must specify `deltaSyncTableName` for Delta Sync configuration.');
+      }
+      return {
+        BaseTableTTL: !config.deltaSyncConfig.baseTableTTL ?
+          0 : config.deltaSyncConfig.baseTableTTL,
+        DeltaSyncTableName: config.deltaSyncConfig.deltaSyncTableName,
+        DeltaSyncTableTTL: !config.deltaSyncConfig.deltaSyncTableTTL ?
```

Comment:
@yyoshiki41 this looks wrong. If you set `0` for `deltaSyncTableTTL` you would get `60`.
Check for `undefined` instead

Reply:
@bboure 
thanks for your review

use `typeof` operator in the below commit.
https://github.com/sid88in/serverless-appsync-plugin/pull/313/commits/2f404d63646510cd94174794f4c765ad5bd1ce04